components/gamification/store/constants.ts

```diff
@@ -1,2 +1,4 @@
-export const TYPE_ELIMINATE_WRONG_ANSWER = 'Eliminate Wrong Answer';
-export const TYPE_DOUBLE_ANSWER = 'Double Answer';
+export const TYPE_ELIMINATE_WRONG_ANSWER = 'Eliminator';
+export const TYPE_ELIMINATE_WRONG_ANSWER_OLD = 'Eliminate Wrong Answer';
+export const TYPE_DOUBLE_ANSWER = 'Double Time';
```

Comment:
Mentioned something related to this above. Also, this feels like something we could do with enums rather than constant strings.